cfme/tests/test_replication.py

```diff
@@ -145,9 +148,9 @@ def test_replication_delete_remote_from_global(setup_replication):
         region.replication.get_replication_status(host=remote_app.hostname)
 
 
-@pytest.mark.manual
+@pytest.mark.provider([OpenStackProvider])
```

Comment:
Remove `@pytest.mark.provider([OpenStackProvider])` Marker as we don't parameterize this test.